nh_food_and_fluid/tests/common/nh_clinical_test_utils.py

```diff
@@ -50,3 +50,24 @@ def create_food_and_fluid_obs_activity(self, fluid_taken=None,
             activity_vals={}, data_vals=obs_data
         )
         return obs_food_and_fluid_activity_id
+
+    def create_f_and_f_review_activity(self, spell_activity=None):
```

Comment:
Could just call `schedule_review` on F&F Review model instead of creating activity. That way that functionality is tested more